artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
I understand...
It is a personal taste, but it is not counterintuitive to have 2 different (and long) properties to be remembered instead of the same?
In addition, consider for example a case in which there is already a 'no' prefix on a property name; the negation of it will risk (depends) to become something like --noNoXXX.